library/src/test/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateAksStepTest.java

```diff
@@ -283,24 +291,26 @@ private void verifyAksNodeResourceGroupName(String actualValue, String managedRe
   }
 
   private void setupParameterResolver() {
-    when(mockParametersResolver.getValue(
-            LandingZoneDefaultParameters.ParametersNames.AKS_MACHINE_TYPE.name()))
-        .thenReturn(ContainerServiceVMSizeTypes.STANDARD_A2_V2.toString());
-    when(mockParametersResolver.getValue(
-            LandingZoneDefaultParameters.ParametersNames.AKS_NODE_COUNT.name()))
-        .thenReturn("1");
-    when(mockParametersResolver.getValue(
-            LandingZoneDefaultParameters.ParametersNames.AKS_AUTOSCALING_ENABLED.name()))
-        .thenReturn("false");
-    when(mockParametersResolver.getValue(
-            LandingZoneDefaultParameters.ParametersNames.AKS_COST_SAVING_SPOT_NODES_ENABLED.name()))
-        .thenReturn(costSavingsSpotNodesEnabled);
-    when(mockParametersResolver.getValue(
-            LandingZoneDefaultParameters.ParametersNames.AKS_COST_SAVING_VPA_ENABLED.name()))
-        .thenReturn(costSavingsVpaEnabled);
-    when(mockParametersResolver.getValue(
-            LandingZoneDefaultParameters.ParametersNames.AKS_AAD_PROFILE_USER_GROUP_ID.name()))
-        .thenReturn("00000000-0000-0000-0000-000000000000");
+    mockParametersResolver =
```

Comment:
Now that `ParametersResolver` is a `record`, it can't be mocked by Mockito anymore. The mocking isn't all that useful anymore anyways since mocking the `getValue` calls one by one is like creating the parameter map one by one. Instead of mocking, just create the `ParametersResolver` with the desired parameter map.

The same change was made in `CreatePostgresqlDbStepTest`, `CreateStorageAccountStepTest`, and `CreateVnetStepTest`